### PR TITLE
Fix typos in docs

### PR DESCRIPTION
**Pull request by NotSimple**
## Checklist

- [x] I have read the [How to Contribute](https://developers.cardano.org/docs/portal-contribute/).
- [x] I have run `yarn build` after adding my changes **without getting any errors**. 

## Fixing typos in the docs

I noticed a few typos in the documention and decided to run [cSpell ](https://cspell.org/) on the docs to find all typos. I've removed everything I could find with a manual revision of what cSpell reported. I've also checked for terms explicitly discouraged in the [editorial guidelines](https://developers.cardano.org/docs/portal-style-guide/#editorial-style-guide) and replaced them with the correct term.

Maybe we could configure cSpell in the CI or a git hook to prevent (or at least reduce) future typos.

### CODE REVIEW

**Review by katomm — APPROVED** [review]
Good contribution, thank you. Also brilliant idea to automate cSpell, we should look into this.

File: docs/operate-a-stake-pool/frankenwallet.md [hunk 1]
```diff
@@ -93,7 +93,7 @@ Though regularly used Frankenwallets have been built on cheap & slow USB drives,
   - a memory stick with a high benchmark for reading <span class="underline">and writing</span> speed, or
   - (for best results in author's experience) a SATA SSD drive plus a SATA-to-USB adapter cable.
 
-Users who have built dual-boot configurations before will also know you should **familiarise yourself with the computer's BIOS settings** in anticipation of the same type of setup.
+Users who have built dual-boot configurations before will also know you should **familiarize yourself with the computer's BIOS settings** in anticipation of the same type of setup.
```
Comment (rphair) [inline docs/operate-a-stake-pool/frankenwallet.md:96]: **Not** a spelling mistake.

**Review by rphair — COMMENTED** [review]
I also love the idea of integrating `cSpell` - but wonder how it would fit in with the GitHub CI, because we couldn't make successful builds contingent upon arbitrary criteria like spelling mistakes.

@fill-the-fill @katomm @rdlrt @os11k what about making it a `yarn` target... which could be done without impacting the GitHub flow, and would expose the spell checking where it really needs to run (the local build, as seen by the contributor)?

@NotSimple your contribution here is excellent & makes the Dev Portal look a lot more professional, but both readers & authors alike would be antagonised by the imposition of USA English in the spell checker, which characterises the valid cases of British English below as spelling mistakes (I double-checked & marked each case so we can count them easily).

Therefore would you please consider refactoring your spell check against a USA English **plus** British English dictionary?  I don't know `cSpell` usage but most dictionaries will additively include spelling dictionaries so you might even tick some other English dialects if allowed (Australia + Canada + India [5 in total] should cover all possible dialect). 😎

File: docs/get-started/air-gap.md [hunk 2]
```diff
@@ -10,7 +10,7 @@ image: /img/og/og-security-air-gap-environment.png
 
 Developers & Cardano stake pool operators generally need an air gap environment in which to work with payment keys, stake pool keys and other cryptocurrency resources that offer high-value targets for hackers.
 
-Some specialised hardware (e.g. hardware wallets) may also perform this function. If you believe you have such a device, please be certain that it offers isolation features for your stake pool or development *and* that you feel secure entrusting your assets to those who have implemented these features.
+Some specialized hardware (e.g. hardware wallets) may also perform this function. If you believe you have such a device, please be certain that it offers isolation features for your stake pool or development *and* that you feel secure entrusting your assets to those who have implemented these features.
```
[the remark this answers is not in the document]
Comment (rphair) [reply docs/get-started/air-gap.md]: fixed by https://github.com/cardano-foundation/developer-portal/pull/1243/commits/17c4088efa18f0df451229af5547d7fa46bca9b4

File: docs/get-started/running-cardano.md [hunk 3]
```diff
@@ -21,7 +21,7 @@ This guide does not cover the topic of running a block-producing `cardano-node`
 ### Testnet
 There are two types of testnet: `preview` and `pre-prod`.
 
-- **Preview Testnet**: Testing release candidates and Mainnet releases. Leads Mainnet hard forks by at least 4 weeks. This net is for those who just want to see how it runs, get familiarised and play with cardano-node.
+- **Preview Testnet**: Testing release candidates and Mainnet releases. Leads Mainnet hard forks by at least 4 weeks. This net is for those who just want to see how it runs, get familiarized and play with cardano-node.
```
[the remark this answers is not in the document]
Comment (rphair) [reply docs/get-started/running-cardano.md]: fixed by https://github.com/cardano-foundation/developer-portal/pull/1243/commits/17c4088efa18f0df451229af5547d7fa46bca9b4

File: docs/get-started/koios.md [hunk 4]
```diff
@@ -31,7 +31,7 @@ If you notice an issue or have a feature request (existing or new endpoint), we
 
 ## Participating as instance provider
 
-There will always be an audience who might want to run everything locally instead of relying on any external connection. Typically, these would be heavy users of the API (explorers/wallet providers/marketplaces), who'd like to override default API limits, have customisation requirements, remove latency hops OR enthusiasts who'd like to contribute back and add to the strength of the API layer.
+There will always be an audience who might want to run everything locally instead of relying on any external connection. Typically, these would be heavy users of the API (explorers/wallet providers/marketplaces), who'd like to override default API limits, have customization requirements, remove latency hops OR enthusiasts who'd like to contribute back and add to the strength of the API layer.
```
[the remark this answers is not in the document]
Comment (rphair) [reply docs/get-started/koios.md]: fixed by https://github.com/cardano-foundation/developer-portal/pull/1243/commits/17c4088efa18f0df451229af5547d7fa46bca9b4

**Comment by NotSimple** [comment]
@rphair I completely understand your point on not imposing USA English. cSpell supports multiple languages and the ability to add custom dictionaries. I adjusted the config to include the "en-GB" locale and removed the unnecessary changes.

File: docs/get-started/cscli.md [hunk 5]
```diff
@@ -7,7 +7,7 @@ image: /img/og/og-getstarted-cscli.png
 --- 
 
 `cscli` is a lightweight cross-platform CLI tool for Cardano supporting the following features out of the box:
- - Building and serialising wallet primitives (i.e. recovery-phrases, keys, addresses and transactions) 
+ - Building and serializing wallet primitives (i.e. recovery-phrases, keys, addresses and transactions)
```
[the remark this answers is not in the document]
Comment (rphair) [reply docs/get-started/cscli.md]: not reverted in https://github.com/cardano-foundation/developer-portal/pull/1243/commits/17c4088efa18f0df451229af5547d7fa46bca9b4 (have reverted it myself in https://github.com/cardano-foundation/developer-portal/pull/1243/commits/d022f37cf1e191d40c27c9386529aebddb3f97b1)

File: docs/native-tokens/authenticated-products.md [hunk 6]
```diff
@@ -16,7 +16,7 @@ For the owner of a POC Hoodie the verification process is as straightforward as
...
+What actually happens in the background when the user taps the NFC tag is that by tapping, the smart phone gets provided with a URL that contains encrypted data that identifies and allows the backend to verify the corresponding NFT on the Cardano blockchain. Following that URL the user ends up on the already mentioned website, which forwards the encrypted data to a validation service that is able to decrypt it and use the encoded information (namely the asset id) to look for a corresponding NFT on the Cardano blockchain. If it finds one, the [metadata of this NFT](https://beta.explorer.cardano.org/en/token/asset1mphtfzam7z23pkzqptpymha6qwm4j6w32275r9/metadata) is analyzed and compared to the data stored on the NFC chip and thus verified. This verification is based on asymmetric key cryptography meaning a signature made during the minting process is verified with the key data sent within the encrypted data.
```
[the remark this answers is not in the document]
Comment (rphair) [reply docs/native-tokens/authenticated-products.md]: fixed by https://github.com/cardano-foundation/developer-portal/pull/1243/commits/17c4088efa18f0df451229af5547d7fa46bca9b4#diff-318af3bbfc78c2d3c1a1809e17dc8f45c0b0e8c42028a00981707e55838b8050

**Review by rphair — APPROVED** [review]
😎👍

**Review by rdlrt — APPROVED** [review]

**Review by os11k — APPROVED** [review]

**Pull request merged**